Functions.Templates/Templates/BlobTrigger-CSharp/BlobTriggerCSharp.csx

```diff
@@ -1,4 +1,4 @@
-public static void Run(Stream myBlob, string name, ILogger log)
+public void Run(Stream myBlob, string name)
 {
     log.LogInformation($"C# Blob trigger function Processed blob\n Name:{name} \n Size: {myBlob.Length} Bytes");

```

Comment:
This would cause compile error. Where is log coming from.

Reply:
I will discuss about the changes needed in .csx files from you and Fabio offline and then update the code here.